javascripts/src/dataclips.js

```diff
@@ -143,14 +143,21 @@ export default class Dataclips {
       });
   }
 
-  downloadXLSX(data, schema, filename) {
+  downloadXLSX(data, schema, filename, disableSeconds) {
+    function formatZeros(value) {
+      if (isNaN(value)) {
+        return '00'
```

Comment:
i think it's wrong. It means that for empty values you are setting '00' where it should be still empty

Reply:
No, because on file w'll have something like 1:23:NaN

Reply:
so something is completely wrong as you should have no seconds, you should cut them off. You should change only values if "disableSeconds" is true, for disableSecond = false the code should stay untouched